connect/runtime/src/main/java/org/apache/kafka/connect/tools/PredicateDoc.java

```diff
@@ -53,29 +52,29 @@ private <P extends Predicate<?>> DocInfo(Class<P> predicateClass, String overvie
         .sorted(Comparator.comparing(docInfo -> docInfo.predicateName))
         .collect(Collectors.toList());
 
-    private static void printPredicateHtml(PrintStream out, DocInfo docInfo) {
-        out.println("<div id=\"" + docInfo.predicateName + "\">");
+    private static void printPredicateHtml(DocInfo docInfo) {
```

Comment:
I wonder if we should instead switch these method to be `toHtml()` like in other module and have them return a String. That way it's testable (I think the idea of passing a `PrintStream` was to make it easily testable) and consistent with other modules. WDYT?

Reply:
Done